examples/sinatra/views/profile.erb

```diff
@@ -1,62 +1,136 @@
-<html>
-<head>
-  <title>Welcome</title>
-</head>
-<body>
-  <h1><a href="/">Home</a></h1>
-  <table>
-    <tr>
-      <td>User ID</td>
-      <td><%= user_id %></td>
-    </tr>
-    <tr>
-      <td>Selfie as base64 URI<br />
-      <td><img src="<%= base64_selfie_uri %>" /></td>
-    </tr>
-    <tr>
-      <td>Selfie as image file<br />
-      <td><img src="selfie.jpeg" /></td>
-    </tr>
-    <tr>
-      <td>Full name</td>
-      <td><%= full_name %></td>
-    </tr>
-    <tr>
-      <td>Given names</td>
-      <td><%= given_names %></td>
-    </tr>
-    <tr>
-      <td>Family name</td>
-      <td><%= family_name %></td>
-    </tr>
-    <tr>
-      <td>Mobile number</td>
-      <td><%= mobile_number %></td>
-    </tr>
-    <tr>
-      <td>Email address</td>
-      <td><%= email_address %></td>
-    </tr>
-    <tr>
-      <td>Date of birth</td>
-      <td><%= date_of_birth %></td>
-    </tr>
-    <tr>
-      <td>Age verified</td>
-      <td><%= age_verified %></td>
-    </tr>
-    <tr>
-      <td>Address</td>
-      <td><%= address %></td>
-    </tr>
-    <tr>
-      <td>Gender</td>
-      <td><%= gender %></td>
-    </tr>
-    <tr>
-      <td>Nationality</td>
-      <td><%= nationality %></td>
-    </tr>
-  </table>
+<%
+attributes = [
+  {
+    "label"=> "Given names",
+    "obj"=> profile.given_names,
+    "icon"=> "yoti-icon-profile"
+  },
+  {
+    "label"=> "Family names",
+    "obj"=> profile.family_name,
+    "icon"=> "yoti-icon-profile"
+  },
+  {
+    "label"=> "Mobile number",
+    "obj"=> profile.phone_number,
+    "icon"=> "yoti-icon-phone"
+  },
+  {
+    "label"=> "Email address",
+    "obj"=> profile.email_address,
+    "icon"=> "yoti-icon-email"
+  },
+  {
+    "label"=> "Date of birth",
+    "obj"=> profile.date_of_birth,
+    "icon"=> "yoti-icon-calendar"
+  },
+  {
+    "label"=> "Age verified",
+    "label_prefix"=> "Age Verification/",
+    "obj"=> profile.age_condition,
+    "icon"=> "yoti-icon-verified"
+  },
+  {
+    "label"=> "Address",
+    "obj"=> profile.postal_address,
+    "icon"=> "yoti-icon-address"
+  },
+  {
+    "label"=> "Gender",
+    "obj"=> profile.gender,
+    "icon"=> "yoti-icon-gender"
+  }
```

Comment:
Are we not displaying `nationality`? Or am I missing it?

Reply:
Good spot, I missed that one.